library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php

```diff
@@ -20,7 +20,7 @@ public static function getPropertyMap(string $context = '', string $role = null)
             $response = parent::getPropertyMap(...func_get_args());
         }
 
-        if ($role === 'ROLE_BRAND_ADMIN') {
+        if (in_array($role, ['ROLE_BRAND_ADMIN', 'ROLE_BRAND_ADMIN'])) {
```

Comment:
Duplicate ROLE_BRAND_ADMIN in array

Reply:
:+1: the second one was ment to be ROLE_COMPANY_ADMIN